library/general/containerresources/suite.yaml

```diff
@@ -0,0 +1,69 @@
+kind: Suite
+apiVersion: test.gatekeeper.sh/v1alpha1
+metadata:
+  name: containerresources
+tests:
+  - name: container-limits-and-requests
+    template: template.yaml
+    constraint: samples/container-must-have-limits-and-requests/constraint.yaml
+    cases:
+      - name: example-allowed1
+        object: samples/container-must-have-limits-and-requests/example_allowed1.yaml
+        assertions:
+          - violations: no
+      - name: example-disallowed1
```

Comment:
nit: I wonder if we could name these tests to explain what they are testing?

Reply:
Renamed tests, but not sure if it's more readable/clear now, do you have any naming conventions?

Reply:
No conventions as yet, since the test function is fairly new, but this is much more clear. Thank you for doing this!